From a2ca7b072ec47cb8d920f0f1b0b6779dcfa375c2 Mon Sep 17 00:00:00 2001
From: Ukri Niemimuukko <ukri.niemimuukko@intel.com>
Date: Wed, 2 Jan 2013 15:43:38 +0200
Subject: [PORT FROM MAIN] fix the typedef for struct camera2_jpeg_blob

BZ: 77791

This patch fixes the typedef for struct camera2_jpeg_blob, which
was missing the typedef name at the end of the structure.

Change-Id: Iae8c8e6d8e889c1d23a0e3a1d56f1276e8a153c5
Orig-Change-Id: Ie5d696c2aec1fede08599e0757786bf912860914
Signed-off-by: Ukri Niemimuukko <ukri.niemimuukko@intel.com>
Reviewed-on: http://android.intel.com:8080/84560
Reviewed-by: Koski, Anttu <anttu.koski@intel.com>
Tested-by: Koski, Anttu <anttu.koski@intel.com>
Reviewed-by: Verdu, Miguel <miguel.verdu@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/hardware/camera2.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/hardware/camera2.h b/include/hardware/camera2.h
index 5b182a8..e5f02e3 100644
--- a/include/hardware/camera2.h
+++ b/include/hardware/camera2.h
@@ -142,7 +142,7 @@ enum {
 typedef struct camera2_jpeg_blob {
     uint16_t jpeg_blob_id;
     uint32_t jpeg_size;
-};
+} camera2_jpeg_blob_t;
 
 enum {
     CAMERA2_JPEG_BLOB_ID = 0x00FF
-- 
1.7.4.1

